ezy-client.js

```diff
@@ -108,6 +144,12 @@ class EzyClient {
         this.connector.connect(this, this.url);
     }
 
+    /**
+     * Client tries to reconnect to previous connection
```

Comment:
`previous connection` không được rõ nghĩa, nên là: Call to connect to server again?